controllers/user/deploy/README.md

```diff
@@ -8,10 +8,16 @@ sealos build -t docker.io/labring/sealos-user-controller:dev -f Dockerfile .
 
 | Name | Description | Default |
 | --- | --- | --- |
-|`PaymentCallbackURL`|Payment callback url|`http://localhost:8080/payment/callback`|
+|`PayCallbackURL`|Payment callback url|``|
+|`WechatPrivateKey`|Wechat private key|``|
+|`MchID`|Wechat mch id|``|
+|`MchCertificateSerialNumber`|Wechat mch certificate serial number|``|
+|`MchAPIv3Key`|Wechat mch api v3 key|``|
+|`AppID`|Wechat app id|``|
 
 ### How to run
 
 ```shell
+sealos run docker.io/labring/kustomize:v4.5.6
```

Comment:
seems we don't need the callback URL anymore.

Reply:
I changed it to `NotifyCallbackURL`. It is actually a callback interface for payment to inform us of the status. I think it's more appropriate to keep it.

Reply:
We don't use Notify too, controller will check the payment status every 0.3s, so needn't any notify

Reply:
> We don't use Notify too, controller will check the payment status every 0.3s, so needn't notify

Reply:
> We don't use Notify too, controller will check the payment status every 0.3s, so needn't any notify

OK，Let me fix this problem.